.gitignore

```diff
@@ -1,3 +1,12 @@
+# Temporary for alternate dev setup
+conf_postgres
+conf_solr
+Dockerfile-defaultworker
+Dockerfile-hydroshare
+Dockerfile-postgis
+Dockerfile-solr
+
+# HydroShare Custom
```

Comment:
Note that the only Dockerfile in use is for hydroshare and is called Dockerfile. The dockerfile-compose.yml is already ignored by the current setup, so conveniently covers my use case.